rotkehlchen/chain/evm/decoding/structures.py

```diff
@@ -67,8 +68,11 @@ class TransferEnrichmentOutput(NamedTuple):
     matched_counterparty is optionally set if needed for enrichment rules
     and is used in post-decoding rules like in the case of balancer
     """
+    enriched: bool = False
     matched_counterparty: Optional[str] = None
+    refresh_balances: bool = False
```

Comment:
In the docstring above you need to explain what the two new arguments are doing here. Separate the docstring in bullet points to explain each argument